src/Uno.UI/Controls/NavigationBarHelper.Android.cs

```diff
@@ -0,0 +1,62 @@
+﻿#if __ANDROID__
+using Android.App;
+using Android.Util;
+using Android.Views;
+using Window = Windows.UI.Xaml.Window;
+
+namespace Uno.UI.Controls
+{
+	public static class NavigationBarHelper
+	{
+		public static bool IsNavigationBarTranslucent
+		{
+			get
+			{
+				var activity = ContextHelper.Current as Activity;
+				var windowFlags = activity.Window.Attributes.Flags;
+
+				return ((int)windowFlags & (int)WindowManagerFlags.TranslucentNavigation) != 0
+					|| ((int)windowFlags & (int)WindowManagerFlags.LayoutNoLimits) != 0;
+			}
+		}
+
+		public static bool IsNavigationBarVisible
+		{
+			get
+			{
+				var activity = ContextHelper.Current as Activity;
+				var systemUiVisibility = activity.Window.DecorView.SystemUiVisibility;
+
+				return (Window.Current.SystemUiVisibility & (int)SystemUiFlags.HideNavigation) == 0
+					|| ((int)systemUiVisibility & (int)SystemUiFlags.HideNavigation) == 0;
+			}
+		}
+
+		public static double LogicalNavigationBarHeight
+		{
+			get
+			{
+				var metrics = new DisplayMetrics();
+				var defaultDisplay = (ContextHelper.Current as Activity)?.WindowManager?.DefaultDisplay;
+
+				if (defaultDisplay == null)
+				{
+					return 0;
+				}
+
+				defaultDisplay.GetMetrics(metrics);
```

Comment:
It would nice to add some comment explaining the difference between `GetMetrics`, and `GetRealMetric`.